music21/stream/__init__.py

```diff
@@ -10567,7 +10556,6 @@ def flattenUnnecessaryVoices(self, *, force=False, inPlace=False):
                     returnObj.coreInsert(shiftOffset + e.getOffsetBySite(v), e)
                 returnObj.remove(v)
 
-        returnObj.coreElementsChanged()
```

Comment:
We need this, because we are doing a coreInsert above.  But it can move to the end of the `if len(flatten) == 1` line.

Reply:
I had a question about this and another -- `stream.remove()` and `stream.insert()` both run `coreElementsChanged` at the end, so the idea in this particular instance is that it's duplicative.

However--looking at the call in `remove`, `clearIsSorted` is set to False -- so, I take it there may be a reason to call it after all, with the default `clearIsSorted=True`. Is that right?